packages/catalog-model/examples/apis/petstore-webhook-api.yaml

```diff
@@ -16,6 +16,6 @@ metadata:
 spec:
   type: openapi
   lifecycle: experimental
-  owner: team-c
```

Comment:
Do we need this change? Can maybe remove this and the changeset for the package?

Reply:
I guess not, it was to test so the sorting worked for a nested key